src/components/WwwFrame/PromotionalBanner/GlobalPromotionalBannerContentful.vue

```diff
@@ -76,10 +60,6 @@ export default {
 		preFetch(config, client) {
```

Comment:
Without the experiment query, we can just switch this back to the default preFetch: 
`query: bannerQuery,
preFetch: true`